internal/dev_server/ui/src/App.tsx

```diff
@@ -21,65 +31,126 @@ function App() {
             }, {});
 
           setFlags(sortedFlags);
+          setOverrides(overrides);
         }
       })
       .catch((e) => {
         // todo: handle failure to load flag data
       });
   }, []);
 
+  const updateOverride = (flagKey: string, overrideValue: LDFlagValue) => {
+    fetch(`api/dev/projects/default/overrides/${flagKey}`, {
+      method: "PUT",
+      body: JSON.stringify({
+        value: overrideValue,
+      }),
+    })
+      .then((res) => {
+        const updatedOverrides = {
```

Comment:
Note: at present we only fetch flag / override values when the page loads... as the user adds / updates / removes overrides, we are making API calls to persist those changes and then updating the in-memory state to get the UI updated.